journals/apps/journals/templates/videos/import_video.html

```diff
@@ -0,0 +1,146 @@
+                $statusElement.text('In Progress');
+                $successElement.html('');
+                $failureElement.html('');
+                e.preventDefault();
+                $workingElement.addClass('icon-spinner');
+                $.post(this.action, $form.serialize(), function(data) {
+                    $successElement.html(data.success_message);
+                    $failureElement.html(data.failure_message);
+                }).fail(function() {
+                    $failureElement.html('{% trans "Failed to import videos."%}');
+                }).always(function() {
+                    $statusElement.text('Completed');
+                    $workingElement.removeClass('icon-spinner');
+                    $btnElement.prop("disabled", false);
+                });
+            });
+        });
+    </script>
+{% endblock %}
+
+{% block content %}
+    {% trans "Import Videos for a journal" as action_str %}
+    {% include "wagtailadmin/shared/header.html" with title=action_str icon="media-full-inverse" %}
+
+    <div class="nice-padding" id="import-videos-container">
+        <div class="drop-zone">
+            <form action="{% url 'journals:import_videos' %}" method="POST">
+                {% csrf_token %}
+                <ul class="fields">
+                    <li class="message">{% trans "Import Videos" %}</li>
+                    {% if journals %}
+                        <li>
+                            <label for="id_import_video_journal">{% trans "Journal:" %}</label>
+                            <div class="field-content">
+                                <select id="id_import_video_journal" name="journal">
+                                    {% for journal in journals %}
+                                        <option value="{{ journal.id }}">{{ journal.name }}</option>
+                                    {% endfor %}
+                                </select>
+                                <p class="help">{% trans "Select a journal to import videos into it" %}</p>
+                            </div>
+                        </li>
+                    {% endif %}
+                    <li>
+                        <label>{% trans "Course runs:" %}</label>
+                        <div class="field-content">
+                            <ul class="field-content course-runs">
+                                <li>
+                                    <input type="checkbox" name="course_runs_all" value="" id="course_run_all">
+                                    <label for="course_run_all">All</label>
+                                    <span class="help">{% trans "Select course runs to find videos from" %}</span>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    <li>
+                        <label for="id_import_video_collection">{% trans "Collection:" %}</label>
```

Comment:
Move this section directly beneath Journal section (line 110)
Change label from `Collection` to `Add to collection`